test/components/tabTest.js

```diff
@@ -63,6 +63,32 @@ describe('tabs', function () {
     })
   })
 
+  describe('close tab', function() {
+    Brave.beforeAll(this)
+    before(function *() {
+      yield setup(this.app.client)
+    })
+    it('should close the tab', function *() {
+      yield this.app.client
+        .ipcSend(messages.SHORTCUT_NEW_FRAME, 'https://www.brave.com')
+        .waitForExist('.tab[data-frame-key="2"]')
+        .ipcSend(messages.SHORTCUT_CLOSE_FRAME)
+
+      yield this.app.client
+        .isExisting('.tab[data-frame-key="2"]').should.eventually.be.false
```

Comment:
I think we've found that should.eventually.be is very intermittent error prone. Could you change those to waitUntil equivalents?